.circleci/config.yml

```diff
@@ -163,3 +176,26 @@ workflows:
     jobs:
       - openwrt_build_ar71xx
       - openwrt_build_ipq806x
+
+  quarterly:
+    triggers:
+      - schedule:
+          cron: "15 8 1 1,4,7,10 *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - check_openwrt_vers:
+          context: $SOME_CONTEXT_WITH_GITHUB_CREDS
+
+  openwrt_bump:
```

Comment:
What was this section for? Not sure I followed what the intent was here for `openwrt_bump`

Reply:
Sure. This just makes the builds run immediatly when the PR is created. That way we can review the build status when we review the PR. 

So when quarterly runs, if there are updated values then the script will create a new branch with something like git checkout -b "openwrt/${TIMESTAMP}". This openwrt_bump job basically only reacts to branches with openwrt/*. It simply runs the same jobs as monthly, in reaction to the automated PR.